projects/testing/src/lib/testing.module.spec.ts

```diff
@@ -1,20 +1,21 @@
-import { TestBed, inject, waitForAsync } from '@angular/core/testing';
-import { TranslateTestingModule } from '../public_api';
```

Comment:
I have run prettier and cleaned up unused imports for each file I touched. Please let me know if I should for some reason revert these changes.